src/test/scala/it/pagopa/interop/catalogprocess/CatalogProcessSpec.scala

```diff
@@ -123,119 +125,119 @@ class CatalogProcessSpec extends SpecHelper with AnyWordSpecLike with BeforeAndA
-        declaredAttributes = List.empty,
-        suspendedByConsumer = None,
-        suspendedByProducer = None,
-        suspendedByPlatform = None,
-        consumerDocuments = List.empty,
-        createdAt = OffsetDateTime.now(),
-        updatedAt = None,
-        consumerNotes = None,
-        stamps = Stamps()
-      )
-
-      (agreementManagementService
-        .getAgreements(_: Option[String], _: Option[String], _: List[AgreementState], _: Option[String])(
-          _: Seq[(String, String)]
-        ))
-        .expects(Some(consumerId.toString), None, List(AgreementState.ACTIVE), None, *)
-        .returning(Future.successful(Seq(activeAgreement)))
-        .once()
-
-      val eservice1 = CatalogManagementDependency.EService(
-        eserviceId1,
-        producerId1,
-        "eservice1",
-        "eservice1",
-        CatalogManagementDependency.EServiceTechnology.REST,
-        CatalogManagementDependency.Attributes(Seq.empty, Seq.empty, Seq.empty),
-        Seq.empty
-      )
-
-      (catalogManagementService
-        .getEService(_: String)(_: Seq[(String, String)]))
-        .expects(eserviceId1.toString, *)
-        .returning(Future.successful(eservice1))
-        .once()
-
-      val tenant = Tenant(
-        id = producerId1,
-        selfcareId = selfcareId1.some,
-        externalId = null,
-        features = Nil,
-        attributes = Nil,
-        createdAt = OffsetDateTimeSupplier.get(),
-        updatedAt = None
-      )
-
-      val org1 =
-        PartyManagementDependency.Institution(
-          UUID.fromString(selfcareId1),
-          "",
-          "",
-          "description1",
-          "",
-          "",
-          "",
-          "",
-          "",
-          institutionType = "",
-          Seq.empty
-        )
-
-      (tenantManagementService
-        .getTenant(_: UUID)(_: Seq[(String, String)]))
-        .expects(producerId1, *)
-        .once()
-        .returning(Future.successful(tenant))
-
-      (partyManagementService
-        .getInstitution(_: String)(_: Seq[(String, String)], _: ExecutionContext))
-        .expects(selfcareId1, *, *)
-        .returning(Future.successful(org1))
-        .once()
-
-      (attributeRegistryManagementService
-        .getAttributesBulk(_: Seq[UUID])(_: Seq[(String, String)]))
-        .expects(Seq.empty, *)
-        .returning(Future.successful(Seq.empty))
-        .once()
-
-      val response = request(s"eservices?consumerId=$consumerId&agreementStates=ACTIVE", HttpMethods.GET)
-
-      response.status shouldBe StatusCodes.OK
-      val body = Await.result(Unmarshal(response.entity).to[Seq[EService]], Duration.Inf)
-      body shouldBe Seq(
-        EService(
-          eserviceId1,
-          Organization(producerId1, "description1"),
-          "eservice1",
-          "eservice1",
-          EServiceTechnology.REST,
-          Attributes(Seq.empty, Seq.empty, Seq.empty),
-          Seq.empty
-        )
-      )
-    }
-  }
+//  "Eservice listing" must {
```

Comment:
this should become an integration test with a running instance of mongodb/remote instance of documentdb, but I didn't want to block tasks depending on this development.
Let's discuss how to proceed